components/library/gobject-introspection/Makefile

```diff
@@ -56,18 +60,13 @@ COMPONENT_PREP_ACTION += \
 CONFIGURE_OPTIONS += --sysconfdir=/etc
 
 CONFIGURE_ENV += PYTHON=$(PYTHON.$(PYTHON_VERSION).$(BITS))
+CONFIGURE_ENV += CPPFLAGS="$(CPPFLAGS)"
 
 COMPONENT_BUILD_ENV += CC="$(CC)"
 
 unexport SHELLOPTS
 
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
-
-test:		$(TEST_32_and_64)
-
-REQUIRED_PACKAGES += library/python/markdown-34
+REQUIRED_PACKAGES += library/python/markdown-35
```

Comment:
This builds and publishes but the resulting package doesn't seem to work as expected.
I have installed the new version on my build server and tried to build libmatekkbd, which depends on it, and its build fails:

In file included from /usr/include/gtk-3.0/gtk/gtkicontheme.h:27:0,
                 from /usr/include/gtk-3.0/gtk/gtk.h:127,
                 from /oibuild/oi-userland/components/desktop/mate/libmatekbd/libmatekbd-1.22.0/libmatekbd/matekbd-keyboard-drawing.c:21:
/usr/include/gtk-3.0/gtk/gtkstylecontext.h:1201:6: note: declared here
 void gtk_style_context_get_background_color (GtkStyleContext *context,
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CCLD     libmatekbdui.la
  GISCAN   Matekbd-1.0.gir
Traceback (most recent call last):
  File "/usr/bin/g-ir-scanner", line 98, in <module>
    from giscanner.scannermain import scanner_main
  File "/usr/lib/amd64/gobject-introspection/giscanner/scannermain.py", line 35, in <module>
    from giscanner.ast import Include, Namespace
  File "/usr/lib/amd64/gobject-introspection/giscanner/ast.py", line 29, in <module>
    from .sourcescanner import CTYPE_TYPEDEF, CSYMBOL_TYPE_TYPEDEF
  File "/usr/lib/amd64/gobject-introspection/giscanner/sourcescanner.py", line 33, in <module>
    from giscanner._giscanner import SourceScanner as CSourceScanner
ImportError: No module named 'giscanner._giscanner'
make[4]: *** [/usr/share/gobject-introspection-1.0/Makefile.introspection:156: Matekbd-1.0.gir] Error 1
make[4]: Leaving directory '/oibuild/oi-userland/components/desktop/mate/libmatekbd/build/amd64/libmatekbd'
make[3]: *** [Makefile:522: all] Error 2
make[3]: Leaving directory '/oibuild/oi-userland/components/desktop/mate/libmatekbd/build/amd64/libmatekbd'
make[2]: *** [Makefile:535: all-recursive] Error 1
make[2]: Leaving directory '/oibuild/oi-userland/components/desktop/mate/libmatekbd/build/amd64'
make[1]: *** [Makefile:442: all] Error 2
make[1]: Leaving directory '/oibuild/oi-userland/components/desktop/mate/libmatekbd/build/amd64'
gmake: *** [/oibuild/oi-userland/make-rules/configure.mk:183: /oibuild/oi-userland/components/desktop/mate/libmatekbd/build/amd64/.built] Error 2